loopchain/utils/__init__.py

```diff
@@ -391,23 +389,55 @@ def parse_target_list(targets: str) -> list:
     return target_list
 
 
-def init_default_key_value_store(store_identity) -> Tuple[KeyValueStore, str]:
+def rename_db_dir(old_store_name: str, store_name: str):
+    """Rename db directory
+    This method is temporary implementation for remove 'ip:port' in db directory name
+    Ths method is useless after rename all nodes done.
+
+    :param old_store_name: old store name
+    :param store_name: new store name
+    :return:
+    """
+    storage_path = Path(conf.DEFAULT_STORAGE_PATH)
+    if os.path.exists(storage_path / store_name):
+        return
+
+    for dirname in os.listdir(storage_path):
+        if dirname == old_store_name:
```

Comment:
I guess that `rename_db_dir` changes db name:
- from: `./.storage/old_store_name`
- to `./.storage/store_name`.

I'm wondering why you try to iterate all of directories in `storage_path`.
...
```python
        # Iteration removed
        src_path = storage_path / old_store_name
        dst_path = storage_path / store_name
        os.rename(src_path, dst_path)
```

Reply:
done!